Makefile-az.mk

```diff
@@ -1,13 +1,14 @@
 AZURE_LOCATION ?= westus2
+COMMON_NAME ?= karpenter2test
```

Comment:
Love, the common name bit. can we keep it as karpenter? Then I will approve

Reply:
sure. I will revert this name into karpenter.